lib/sensors/thermistor.h

```diff
@@ -0,0 +1,22 @@
+#ifndef THERMISTOR_H
+#include THERMISTOR_H
+#include "mbed.h"
+#include "math.h"
+
+class thermistor{
+    private:
+        constexpr float B = 4700; //kelvins. Based on Resistances of 25 and 50 , part number ERT-J1VV104J

```

Comment:
for all these members except ROOM_TEMP should be passed in from the constructor rather than hardcoded. This is a class for thermistor, not just ERT-J1VV104J. I would also recommend to rename the class sth like VoltageDividerThermistor or add comment that this is a class for thermistors that works by using a voltage divider